deltatech_followup/views/res_partner.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
+        <record id="view_partner_form_followup" model="ir.ui.view">
+            <field name="name">res.partner.followup.inherit</field>
+            <field name="model">res.partner</field>
+            <field name="inherit_id" ref="base.view_partner_form" />
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='category_id']" position="after">
+                    <field name="send_followup" />
+                </xpath>
```

Comment:
Nu este mai bine ca acest camp sa fie in tabul de facturare ?

Reply:
In general la useri li-e lene sa mai dea un click pe tab...